core/src/test/scala/org/apache/spark/ShuffleSuite.scala

```diff
@@ -483,6 +483,17 @@ abstract class ShuffleSuite extends SparkFunSuite with Matchers with LocalRootDi
     }
     assert(e.getMessage.contains("corrupted due to DISK_ISSUE"))
   }
+
+  test("SPARK-39771: warn when shuffle block number is too large") {
+    sc = new SparkContext("local", "test", conf)
+    val logAppender = new LogAppender("deprecated Avro option 'ignoreExtension'")
```

Comment:
`"deprecated Avro option 'ignoreExtension'"` seems to be wrong to me. Is this intentional, @y-wei ?

Reply:
Maybe, is this simply a copy-and-paste mistake from here?
https://github.com/apache/spark/blob/1a408033daf458f1ceebbe14a560355a1a2c0a70/connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala#L2229

Reply:
True, thanks for catching this

Reply:
Haha 😄. I also missed it, I was so focused on the assertion message that completely overlooked the logger.